django/cantusdb_project/main_app/forms.py

```diff
@@ -214,3 +214,50 @@ class Meta:
         choices=TRUE_FALSE_CHOICES_INVEN,
     )
     complete_inventory.widget.attrs.update({"class": "form-control custom-select custom-select-sm"})
+
+class EditChantForm(forms.ModelForm):
```

Comment:
Can you change the class name to `ChantEditForm`? Just to stay consistent with the existing naming schema, like `ChantCreateForm` and `SourceCreateForm` that we already have.